backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -87,9 +87,10 @@ func (fs *experimentStore) CreateExperiment(ctx context.Context, config *any.Any
 			INSERT INTO experiment_run (
                 id, 
 		        experiment_config_id,
-		        execution_time,
+						execution_time,
+						scheduled_end_time,
```

Comment:
nit: indentation doesn't match

(unfortunately i don't know of a way to automatically lint these sql statements with how we're using them currently)